app/models/registration/requests/IndividualWithoutId.scala

```diff
@@ -42,9 +42,18 @@ object IndividualWithoutId {
     (
       answers.getEither(IndividualNamePage),
       answers.getEither(DateOfBirthPage),
-      getAddress(answers)
-    ).parMapN { (name, dateOfBirth, address) =>
-      IndividualWithoutId(name.firstName, name.lastName, dateOfBirth, address)
+      getAddress(answers),
+      answers.getEither(IndividualEmailAddressPage)
+    ).parMapN { (name, dateOfBirth, address, email) =>
+      IndividualWithoutId(
+        name.firstName,
+        name.lastName,
+        dateOfBirth,
+        address,
+        ContactDetails(
+          email,
+          answers.get(IndividualPhoneNumberPage)
```

Comment:
This should not be there, it should be in the same sections where the other similar operations are done.

Reply:
Moved.